engine/common/src/main/java/org/enso/common/ContextFactory.java

```diff
@@ -165,6 +166,7 @@ public Context build() {
         Context.newBuilder()
             .allowExperimentalOptions(true)
             .allowAllAccess(true)
+            .allowIO(IOAccess.ALL)
```

Comment:
What is this good for? I don't think our JavaScript is performing any I/O by itself! The JavaScript code calls into the `ydoc-polyfill` Java classes and only then the Java classes do some I/O - we shouldn't need a GraalVM permission for that.

Reply:
Removed in 557bd008390bc1c03c781f9f0e85c8fc4a195171